app/models/project_image.rb

```diff
@@ -1,3 +1,4 @@
+#FIXME: remove this after the migration has been done and modify the migration accordingly
 class ProjectImage < ActiveRecord::Base
```

Comment:
The migration was done? I don't believe it was done under production.

Em qui, 26 de fev de 2015 às 13:45, Diego de Araújo Martinez Camarinha <
notifications@github.com> escreveu:

> In app/models/project_image.rb
> <https://github.com/mezuro/prezento/pull/183#discussion_r25442336>:
>
> > @@ -1,3 +1,4 @@
> > +#FIXME: remove this after the migration has been done and modify the migration accordingly
> >  class ProjectImage < ActiveRecord::Base
>
> Can we remove this now?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mezuro/prezento/pull/183/files#r25442336>.
>